ui/components/app/beta-header/beta-header.stories.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { Provider } from 'react-redux';
+import testData from '../../../../.storybook/test-data';
+import configureStore from '../../../store/store';
+import BetaHomeHeader from '.';
+
+const store = configureStore({
+  ...testData,
+  metamask: { ...testData.metamask, isUnlocked: true, showBetaHeader: true },
+});
+
+export default {
+  title: 'Components/App/BetaHomeHeader',
+  decorators: [(story) => <Provider store={store}>{story()}</Provider>],
+  id: __filename,
+};
+
+export const DefaultStory = () => (
+  <>
+    <BetaHomeHeader />
```

Comment:
Nit: It's called `BetaHeader` elsewhere

```suggestion
import BetaHeader from '.';

const store = configureStore({
  ...testData,
  metamask: { ...testData.metamask, isUnlocked: true, showBetaHeader: true },
});

export default {
  title: 'Components/App/BetaHeader',
  decorators: [(story) => <Provider store={store}>{story()}</Provider>],
  id: __filename,
};

export const DefaultStory = () => (
  <>
    <BetaHeader />
```

Reply:
Fixed!